### PR TITLE
Remove `mozilla_monitor` conditionals

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
Removed the conditional since Mozilla Monitor has been out for quite some time now, and it's time to retire any of the old and obsolete Firefox Monitor strings and references.

Sibling PR for switch: https://github.com/mozmeao/www-config/pull/560

## Significant changes and points to review

- [ ] Removed conditionals from relevant HTML files
- [ ] Removed obsolete Firefox Monitor strings from relative Fluent files

## Issue / Bugzilla link
fixes https://github.com/mozilla/bedrock/issues/14492

## Testing

- [ ] http://localhost:8000/en-US/account/
- [ ] http://localhost:8000/en-US/firefox/welcome/1/
- [ ] http://localhost:8000/en-US/firefox/welcome/7
- [ ] http://localhost:8000/en-US/firefox/welcome/8
- [ ] http://localhost:8000/en-US/privacy/products/
- [ ] http://localhost:8000/en-US/products/
- [ ] http://localhost:8000/en-US/firefox/privacy/safe-passwords/
- [ ] http://localhost:8000/en-US/firefox/browsers/best-browser/
- [ ] http://localhost:8000/en-US/firefox/privacy/book/

### CODE REVIEW

File: l10n/en/brands.ftl [hunk 1]
```diff
@@ -69,9 +69,6 @@
 -brand-name-firefox-devtools = Firefox DevTools
 -brand-name-firefox-lockwise = Firefox Lockwise
 
-# Obsolete brand name
--brand-name-firefox-monitor = Firefox Monitor
```
Comment (alexgibson) [inline l10n/en/brands.ftl:73]: **Issue:** I still see this brand name variable used in some obsolete strings. We should keep this variable around as long as those strings still reference it?

**Review by alexgibson — CHANGES_REQUESTED** [review]
Good cleanup! Just one small issue below r+wc

**Comment by alexgibson** [comment]
Needs a rebase since https://github.com/mozilla/bedrock/pull/14478 merged

File: l10n/en/brands.ftl (hunk 1 above)
Comment (reemhamz) [reply l10n/en/brands.ftl:73]: Seems all the obsolete strings using this brand name variable are 7+ months old, so I'm pretty sure it's safe to remove them :) i'll work on that now. Thanks for catching this!!! 🦅 👁️

**Review by alexgibson — APPROVED** [review]
r+

I'll leave this one to you to merge when you're ready, nice cleanup work!

**Comment by alexgibson** [comment]
Replaced by https://github.com/mozilla/bedrock/pull/14534

**Pull request closed without merging**